eva_sub_cli/validators/validator.py

```diff
@@ -445,6 +457,52 @@ def _convert_metadata_attribute(self, sheet, json_attribute, xls2json_conf):
             if attributes_dict[attribute] == json_attribute:
                 return attribute
 
+    def _collect_md5sum_to_metadata(self):
+        md5sum_file = resolve_single_file_path(os.path.join(self.output_dir, 'md5sums.txt'))
+        file_path_2_md5 = {}
+        file_name_2_md5 = {}
+        if md5sum_file:
+            with open(md5sum_file) as open_file:
+                for line in open_file:
+                    sp_line = line.split()
+                    md5sum = sp_line[0]
+                    vcf_file = sp_line[1]  # assuming no space in the vcf_file_name
```

Comment:
Not sure this is a safe assumption, if this is the user's full path to the file...

Reply:
I changed it